third_party/nvidia/backend/driver.c

```diff
@@ -279,8 +279,20 @@ static PyObject *setPrintfFifoSize(PyObject *self, PyObject *args) {
   Py_RETURN_NONE;
 }
 
+typedef struct {
+  PyObject_HEAD CUtensorMap tensorMap;
```

Comment:
NIT: I think we can get clang-format to play ball with a semi-colon? Otherwise I would use `// clang-format off`
```suggestion
  PyObject_HEAD;
  CUtensorMap tensorMap;
```